pkg/middleware/healthcheck.go

```diff
@@ -41,8 +41,10 @@ func isHealthCheckRequest(paths, userAgents map[string]struct{}, req *http.Reque
 	if _, ok := paths[req.URL.EscapedPath()]; ok {
 		return true
 	}
-	if _, ok := userAgents[req.Header.Get("User-Agent")]; ok {
-		return true
+	if len(req.Header.Get("User-Agent")) > 0 {
```

Comment:
This no longer makes sense if the healthcheck middleware is responsible for removing the blank entries. The reason for this is that I didn't 'trust' `main.go` prevent construction of the middleware without blank entries since this had no tests enforcing it.